apps/studio/components/interfaces/Organization/TeamSettings/UpdateRolesPanel/UpdateRolesPanelOld.tsx

```diff
@@ -1,232 +0,0 @@
-import { ExternalLink } from 'lucide-react'
```

Comment:
Can this file be deleted?

Reply:
yup! this one's no longer being used as it's the old org team role management panel before project level permissions